apps/condo/pages/_app.tsx

```diff
@@ -73,62 +73,64 @@ const MenuItems: React.FC = () => {
-                path={'/employee'}
-                icon={UserIcon}
-                label={'menu.Employees'}
-                disabled={disabled}
-                isCollapsed={isCollapsed}
-            />
-            <MenuItem
-                path={'/meter'}
-                icon={MeterLog}
-                label={'menu.Meters'}
-                disabled={disabled}
-                isCollapsed={isCollapsed}
-            />
-            <MenuItem
-                path={'/billing'}
-                icon={ApiFilled}
-                label={'menu.Billing'}
-                disabled={disabled}
-                isCollapsed={isCollapsed}
-            />
-            <MenuItem
-                path={'/settings'}
-                icon={SettingFilled}
-                label={'menu.Settings'}
-                disabled={!link}
-                isCollapsed={isCollapsed}
-            />
+            <div>
+                <MenuItem
+                    path={'/reports'}
+                    icon={BarChartIcon}
+                    label={'menu.Analytics'}
+                    disabled={disabled}
+                    isCollapsed={isCollapsed}
+                />
+                <MenuItem
+                    path={'/ticket'}
+                    icon={ThunderboltFilled}
+                    label={'menu.ControlRoom'}
+                    disabled={disabled}
+                    isCollapsed={isCollapsed}
+                />
+                <MenuItem
+                    path={'/property'}
+                    icon={HomeFilled}
+                    label={'menu.Property'}
+                    disabled={disabled}
+                    isCollapsed={isCollapsed}
+                />
+                <MenuItem
+                    path={'/contact'}
+                    icon={UserIcon}
+                    label={'menu.Contacts'}
+                    disabled={disabled}
+                    isCollapsed={isCollapsed}
+                />
+                <MenuItem
+                    path={'/employee'}
+                    icon={UserIcon}
+                    label={'menu.Employees'}
+                    disabled={disabled}
+                    isCollapsed={isCollapsed}
+                />
+                <MenuItem
+                    path={'/meter'}
+                    icon={MeterLog}
+                    label={'menu.Meters'}
+                    disabled={disabled}
+                    isCollapsed={isCollapsed}
+                />
+                <MenuItem
+                    path={'/billing'}
+                    icon={ApiFilled}
+                    label={'menu.Billing'}
+                    disabled={disabled}
+                    isCollapsed={isCollapsed}
+                />
+                <MenuItem
+                    path={'/settings'}
+                    icon={SettingFilled}
+                    label={'menu.Settings'}
+                    disabled={!link}
+                    isCollapsed={isCollapsed}
+                />
+            </div>
```

Comment:
Guess we shouldnt remove any fragment from this component.

Reply:
no, thats not, i've checked